frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -136,6 +136,22 @@ class SceneController {
       return cube;
     };
 
+    window.addVoxelMesh = (position: Vector3, cubeLength: Vector3, optColor?: string) => {
```

Comment:
I used this for debugging purposes, but there seems to be a slight offset in the position. Not sure why, though.

Reply:
Ah, now the penny has dropped :smile: However, the `0.9` is not fully responsible because I saw the offset already before I added the 0.9 factor. As you correctly point out, the flooring is messing things up. Since I used a cube length of `[1, 1, 1]`, the division by two (which is necessary since threejs expects the position of the cube to be in the center) produced fractional numbers which were important for rendering at the correct position, but were removed again by the flooring. I don't know why I even added the flooring in the first place. Maybe I thought, that there would be problems with fractionals :shrug:
I removed the flooring and also explained the 0.9 factor (which I added so that one can see the border between voxels in form of a gap) which wasn't used when instantiating the box geometry, by the way. I guess, it was late when I added this code :sweat_smile: